cli/command/cli.go

```diff
@@ -137,7 +142,28 @@ func (cli *DockerCli) ServerInfo() ServerInfo {
 
 // ClientInfo returns the client details for the cli
 func (cli *DockerCli) ClientInfo() ClientInfo {
-	return cli.clientInfo
+	if cli.clientInfo != nil {
+		return *cli.clientInfo
+	}
+	var experimentalValue string
+	// Environment variable always overrides configuration
+	if experimentalValue = os.Getenv("DOCKER_CLI_EXPERIMENTAL"); experimentalValue == "" {
+		experimentalValue = cli.ConfigFile().Experimental
+	}
+	hasExperimental, _ := isEnabled(experimentalValue)
+
+	version := ""
```

Comment:
this clashes with the `github.com/docker/cli/cli/version` import

```suggestion
	v := ""
```